test/e2e/multi-node.sh

```diff
@@ -25,41 +26,84 @@ export COUCH_URL=http://$user:$password@couchdb-1.local:$COUCH_PORT
 
 # launch cht 4.x CouchDb cluster
-docker-compose -f ./scripts/couchdb-cluster.yml up -d
-docker-compose -f ../docker-compose-test.yml run couch-migration check-couchdb-up 3
+docker compose -f ./scripts/couchdb-cluster.yml up -d
+docker compose -f ../docker-compose-test.yml run couch-migration check-couchdb-up 3
 
 # generate shard matrix
 # this is an object that assigns every shard to one of the nodes
-shard_matrix=$(docker-compose -f ../docker-compose-test.yml run couch-migration generate-shard-distribution-matrix)
+shard_matrix=$(docker compose -f ../docker-compose-test.yml run couch-migration generate-shard-distribution-matrix)
 file_matrix="{\"couchdb@couchdb-1.local\":\"$couch1dir\",\"couchdb@couchdb-2.local\":\"$couch2dir\",\"couchdb@couchdb-3.local\":\"$couch3dir\"}"
 echo $shard_matrix
 echo $file_matrix
 # moves shard data files to their corresponding nodes, according to the matrix
-docker-compose -f ../docker-compose-test.yml run couch-migration shard-move-instructions $shard_matrix
-node ./scripts/distribute-shards.js $shard_matrix $file_matrix
+docker compose -f ../docker-compose-test.yml run couch-migration shard-move-instructions $shard_matrix
+node ./scripts/distribute-shards.js "$shard_matrix" "$file_matrix"
+
 # change database metadata to match the shard physical locations
-docker-compose -f ../docker-compose-test.yml run couch-migration move-shards $shard_matrix
-docker-compose -f ../docker-compose-test.yml run couch-migration verify
+docker compose -f ../docker-compose-test.yml run couch-migration move-shards $shard_matrix
+# Remove old node from cluster
+docker compose -f ../docker-compose-test.yml run couch-migration remove-node couchdb@127.0.0.1
+
+docker compose -f ../docker-compose-test.yml run couch-migration verify
 # test that data exists, database shard maps are correct and view indexes are preserved
 node ./scripts/assert-dbs.js $jsondataddir $shard_matrix
 
-docker-compose -f ./scripts/couchdb-cluster.yml down --remove-orphans --volumes
+# Let's get the shard mapping - we need this to migrate to a different cluster.
+shard_mapping=$(docker compose -f ../docker-compose-test.yml run couch-migration get-shard-mapping)
+echo $shard_mapping
+
+docker compose -f ./scripts/couchdb-cluster.yml down --remove-orphans --volumes
+
+# launch a different cht 4.x CouchDb cluster
+docker compose -f ./scripts/couchdb-cluster-2.yml up -d
+
+# set new COUCH_URL for cluster-2
+export COUCH_URL=http://$user:$password@couchdb-1-namespace-svc-cluster.local:$COUCH_PORT
+
+docker compose -f ../docker-compose-test.yml run couch-migration check-couchdb-up 3
+
+# change database metadata to match new node name
+# Move nodes one by one using move-node.js oldNode:newNode
+
+# Using env variables
+docker compose -f ../docker-compose-test.yml run -e SHARD_MAPPING="$shard_mapping" couch-migration move-node '{"couchdb@couchdb-1.local":"couchdb@couchdb-1-namespace-svc-cluster.local"}'
```

Comment:
There are two tests here that use enb variables. One test should probably use a param.